src/Symfony/Component/Security/Http/Firewall/ContextListener.php

```diff
@@ -76,7 +76,7 @@ public function __construct(TokenStorageInterface $tokenStorage, iterable $userP
      */
     public function setLogoutOnUserChange($logoutOnUserChange)
     {
-        $this->logoutOnUserChange = (bool) $logoutOnUserChange;
+        // no-op, to be deprecated in 4.1
```

Comment:
should it throw if false is provided?

Reply:
Not sure, in the deprecation message of the related option we tell the user that it will be ignored in 4.0. So, I think we can do the same with the method.

Reply:
On v3.0, in ChoiceType, we threw an exception when "$choices_as_values !== true"  in a similar case. Is there any difference here?